integration/quota/quota_test.go

```diff
@@ -136,11 +136,7 @@ func runRateLimitingTest(ctx context.Context, s *testServer, numTokens int) erro
 	if err != nil {
 		return fmt.Errorf("InitLog() returned err = %v", err)
 	}
-	hasherFn, err := trees.Hash(tree)
-	if err != nil {
-		return fmt.Errorf("trees.Hash()=%v, want: nil", err)
-	}
-	hasher := hasherFn.New()
+	hasher := crypto.SHA256.New()
```

Comment:
Should we still verify that `tree.HashAlgorithm == sigpb.DigitallySigned_SHA256`, or it's done elsewhere?

Reply:
Ah, no need to do in the in the tests. What about the main code?

Reply:
This is what I figured (that it was ok in the tests)... At the one "real" call site (`Signer`, in `trees.go`), I replaced the call with a check that it is the expected algorithm.